docs/openapi.yaml

```diff
@@ -11,6 +11,59 @@ info:
     * **from** - starting position of result
     * **results** - integer number of results returned
 paths:
+  /{index}/search:
```

Comment:
Copy-pasted from other search method. At this point we have not specified what the contract should be.